models/project/project.go

```diff
@@ -44,6 +44,9 @@ const (
 
 	// TypeOrganization is a project that is tied to an organisation
 	TypeOrganization
+
+	// TypeUser is a project that is tied to a user
```

Comment:
What's the different from TypeIndividual ?

Reply:
I didn't understand the meaning of `TypeIndividual` at that time, since it is only used here and `ctx.Data["OpenProjects"]` is not used in template.
https://github.com/go-gitea/gitea/blob/e314c9e86656333e9ec956f0d825ef920f491de6/routers/web/user/profile.go#L227-L231

After checking `UserType`, I know now, we have `Individual` and `Organization` user type.
I will remove `TypeUser` and check the code mentioned above.